app/frontend/src/components/designer/settings/FormEventStreamSettings.vue

```diff
@@ -0,0 +1,281 @@
+<script setup>
+import moment from 'moment';
+import { storeToRefs } from 'pinia';
+import { useI18n } from 'vue-i18n';
+import { onMounted, ref } from 'vue';
+
+import { useAppStore } from '~/store/app';
+import { useFormStore } from '~/store/form';
+import { useNotificationStore } from '~/store/notification';
+import { encryptionKeyService } from '~/services';
+
+const { t, locale } = useI18n({ useScope: 'global' });
+const appStore = useAppStore();
+const formStore = useFormStore();
+const notificationStore = useNotificationStore();
+const { form, isRTL } = storeToRefs(formStore);
+
+const loading = ref(true);
+const algorithms = ref([]);
+const techdocsLink =
+  'https://developer.gov.bc.ca/docs/default/component/chefs-techdocs/Capabilities/Integrations/Event-Stream-Service/';
+
+onMounted(async () => {
+  await fetchAlgorithms();
+});
+
+async function fetchAlgorithms() {
+  loading.value = true;
```

Comment:
Does the loading variable need to be used? We do use the skeleton loader sometimes, or we've used the loading variable to block user interaction until loading is done.

Reply:
can you point me at something more current and I can follow that pattern?

Reply:
just going to remove this...